lab-davide/model/task.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const taskSchema = mongoose.Schema({
+  desc: {type: String, required: true},
```

Comment:
Shouldn't tasks belong to notes? 

We want the second resource to have a many-to-one relationship with the first resource.